### PR TITLE
[TW-90] Rename VictorOps to Splunk On-Call

### PR DESCRIPTION
**Pull request by jkonrath-postman**
* Rename page
* Add redirect
* Change names
* Cleaned up to make numbered procedures
* Image changes

TODO:
* Image Changes of Splunk On-Call interface
* Steps may be different on their end to get keys etc.

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/splunk-on-call.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+warning: false
+contextual_links:
+  - type: section
+    name: "Prerequisites"
+  - type: link
+    name: "Grouping requests in collections"
+    url: "/docs/sending-requests/intro-to-collections/"
+  - type: link
+    name: "Intro to Monitoring"
+    url: "/docs/designing-and-developing-your-api/monitoring-your-api/intro-monitors/"
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/splunk-on-call.md:14]: Can we add "search_keyword" too here?

File: src/pages/docs/integrations/available-integrations/splunk-on-call.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+   [![enter key](https://assets.postman.com/postman-docs/58842547.png)](https://assets.postman.com/postman-docs/58842547.png)
+
+## Configuring Postman Monitors
+
+1. From the [Integrations search page](https://go.postman.co/integrations/browse?category=all) search and select Splunk On-Call from the results.
+1. Select **Add Integration** to start configuring this integration for your workspace.
+1. Select a monitor to send to Splunk On-Call.
+1. Enter the Splunk On-Call API key.
+1. Click "Advanced Options" if you want to add a Splunk On-Call Routing Key.
+1. Click the **Add Integration** button.
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/splunk-on-call.md:50]: Click **Add Integration**.

File: src/pages/docs/integrations/available-integrations/splunk-on-call.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+This integration allows you to configure [Postman Monitors](/docs/designing-and-developing-your-api/monitoring-your-api/intro-monitors/) to trigger incidents on Splunk On-Call whenever it fails.
+
+## Get the Splunk On-Call API Key
+
+1. Log in to your Splunk On-Call account and go to the **Settings** tab.
+1. On the **Settings** page, go to the the **Alert Behavior** menu and select **Integrations**.
+   [![log in to Splunk On-Call](https://assets.postman.com/postman-docs/58842896.png)](https://assets.postman.com/postman-docs/58842896.png)
+1. From the list of integrations, select the **REST Generic** Integration.
+   [![select REST Generic](https://assets.postman.com/postman-docs/58843113.png)](https://assets.postman.com/postman-docs/58843113.png)
+1. If the integration is not already enabled, click the *Enable Integration* button.
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/splunk-on-call.md:30]: click the *Enable Integration* button. > "click **Enable Integration**."

File: src/pages/docs/integrations/available-integrations/splunk-on-call.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+> **[Integrations are available on Postman Team, Business, and Enterprise plans.](https://www.postman.com/pricing/)**
+
+Splunk On-Call (formerly VictorOps) is a real-time incident management platform that combines the power of people and data to handle incidents as they occur and prepare for the next one.
+
+This integration allows you to configure [Postman Monitors](/docs/designing-and-developing-your-api/monitoring-your-api/intro-monitors/) to trigger incidents on Splunk On-Call whenever it fails.
+
+## Get the Splunk On-Call API Key
+
+1. Log in to your Splunk On-Call account and go to the **Settings** tab.
+1. On the **Settings** page, go to the the **Alert Behavior** menu and select **Integrations**.
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/splunk-on-call.md:26]: **Alert Behavior** tab maybe?

File: src/pages/docs/integrations/available-integrations/splunk-on-call.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+   You can see your newly created integration and every instance of this integration created by your team.
+
+   ![Splunk On-Call add integration](https://assets.postman.com/postman-docs/splunk-on-call-view-all-integrations.jpg)
+
+1. Click the name of the monitor you added to this integration to view the monitoring results.
+
+## Incidents on Splunk On-Call
+
+An incident on Splunk On-Call consists of the basic information of which monitor failed, with the number of errors and failed tests. It also provides a direct link to the failing monitor. This integration automatically acknowledges a triggered incident if the subsequent run succeeds.
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/splunk-on-call.md:63]: Just a suggestion - Can we reword the first line, this part to be specific "consists of the basic information of which monitor failed" to something like - 
"consists of the basic information about the specific/particular monitor which failed"

File: src/pages/docs/integrations/available-integrations/splunk-on-call.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+   If the integration is enabled, you will see a URL which contains the API Key.
+   [![view API key](https://assets.postman.com/postman-docs/58843264.png)](https://assets.postman.com/postman-docs/58843264.png)
+1. Copy the API Key and save it for later.
+
+You can also provide routing keys if you are using teams within Splunk On-Call. Routing keys allow the creation of incidents to be directed towards a specific team, so that they get notified about a particular failure. To get the routing key:
+
+1. Click on the **Alert Behavior** menu and select *Route Keys*.
+   [![make a note of keys](https://assets.postman.com/postman-docs/58842580.png)](https://assets.postman.com/postman-docs/58842580.png)
+1. Enter your own key and select a team for which the key is applicable.
+   [![enter key](https://assets.postman.com/postman-docs/58842547.png)](https://assets.postman.com/postman-docs/58842547.png)
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/splunk-on-call.md:41]: Also, are we using `.png`formats for the images?

File: redirects.json [hunk 7]
```diff
@@ -1818,4 +1818,8 @@
 		"from": "/docs/publishing-your-api/api-submission-guidelines/",
 		"to": "/docs/publishing-your-api/authoring-your-documentation/"
 }
+{
+		"from": "/docs/integrations/available-integrations/victorops/",
+		"to": "/docs/integrations/available-integrations/splunk-on-call/"
+}
```
Comment (sowmyaraj92) [inline redirects.json:1824]: Can we check if the redirects.json file is working fine?
 I got this error when I tried to run the branch locally - `Unexpected token { in JSON at position 60734`

**Review by sowmyaraj92 — CHANGES_REQUESTED** [review]

**Comment by jkonrath-postman** [comment]
Doing this without the screenshots, just to get the rename and text changes in there.

I'll pick up screenshot changes in tw-107.

**Pull request merged**